GitCommands/Git/GitModule.cs

```diff
@@ -3066,9 +3066,9 @@ public GitBlame Blame(string? fileName, string from, Encoding encoding, string?
             GitArgumentBuilder args = new("blame")
             {
                 "--porcelain",
-                { AppSettings.DetectCopyInFileOnBlame, "-M" },
-                { AppSettings.DetectCopyInAllOnBlame, "-C" },
-                { AppSettings.IgnoreWhitespaceOnBlame, "-w" },
+                { AppSettings.DetectCopyInFileOnBlame, "-M" }, // as git-diff --find-renames
+                { AppSettings.DetectCopyInAllOnBlame, "-C" }, // as git-diff --find-copies
+                { AppSettings.IgnoreWhitespaceOnBlame, "-w" }, // as git-diff --ignore-all-space
```

Comment:
For those who wonder (and have not read the comments in the next file yet): "git blame" does not support the speaking / long arguments.

Reply:
AppSettings name is hopefully mostly describing what is done. 
One reason long options are in comments is to simplify searches

Reply:
> AppSettings name is hopefully mostly describing what is done.

Yes, naming is good enough for me and I was wondering why exactly the comments but the explanation given makes perfectly sense:

> One reason long options are in comments is to simplify searches

I just wonder if the settings property names could be (again) improved a little more:

* `DetectCopyInFileOnBlame` to `DetectCopyWithinFileOnBlame`
* `DetectCopyInAllOnBlame` to `DetectCopyInCommitOnBlame` or `DetectCopyInAllCommitFilesOnBlame`

Point of view?

Reply:
Not changed in this PR